dbmanager/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM ubuntu:18.04
+
+RUN apt-get -y update
+RUN apt-get install -y postgresql-client
+RUN apt-get install -y python3
+RUN mkdir /app
+RUN mkdir /app/current_migration
+
+VOLUME /app/current_migration
```

Comment:
do you think it might make more sense to put this in `docker-compose.yml`?

Reply:
I'm not sure @jazevedo620 what do you think

Reply:
I think we should keep it here imo because no other service is using this volume, so we should keep it contained here.